stable_baselines3/common/torch_layers.py

```diff
@@ -172,11 +177,10 @@ def __init__(
         # Iterate through the shared layers and build the shared parts of the network
         for idx, layer in enumerate(net_arch):
             if isinstance(layer, int):  # Check that this is a shared layer
-                layer_size = layer
                 # TODO: give layer a meaningful name
-                shared_net.append(nn.Linear(last_layer_dim_shared, layer_size))
+                shared_net.append(nn.Linear(last_layer_dim_shared, layer))  # add linear of size layer
```

Comment:
why this change?
to avoid one assignment?

Reply:
Pretty much yea. I'm indifferent to it :p